schema/metrics_config.schema.json

```diff
@@ -57,36 +57,36 @@
                 },
                 "asset-name": {
                     "type": "string",
-                    "description": "Enter the name for a CDF asset that will have all the metrics time series attached to it."
+                    "description": "Enter the name for a CDF asset with all the metrics time series attached to it."
```

Comment:
The asset might not yet exist, we will set all the time series on it, which why it was written in the tense it was